tests/test_connection.py

```diff
@@ -7,7 +7,10 @@
 
 class TestPercyConnection(unittest.TestCase):
     def setUp(self):
-        self.percy_connection = connection.Connection(percy.Config(access_token='foo'))
+        self.percy_connection = connection.Connection(
+            percy.Config(access_token='foo'),
+            "Percy/v1 percy-client/0.4.5 (django/1.9.4; python/2.7.13)"
```

Comment:
Could pass these in as keyword arguments for more clarity (and also personally I would drop the user agent passing in if possible, since we don't really care about it for these tests):

```python
config=percy.Config(access_token='foo'),
user_agent=None,
```